src/test/java/org/kohsuke/github/AbstractGHAppInstallationTest.java

```diff
@@ -89,17 +114,28 @@ private String createJwtToken(String keyFileResouceName, String appId) {
      *             Signals that an I/O exception has occurred.
      */
     protected GHAppInstallation getAppInstallationWithToken(String jwtToken) throws IOException {
+        if (jwtToken.startsWith("Bearer ")) {
+            jwtToken = jwtToken.substring("Bearer ".length());
+        }
+
         GitHub gitHub = getGitHubBuilder().withJwtToken(jwtToken)
                 .withEndpoint(mockGitHub.apiServer().baseUrl())
                 .build();
 
-        GHAppInstallation appInstallation = gitHub.getApp()
-                .listInstallations()
-                .toList()
-                .stream()
-                .filter(it -> it.getAccount().login.equals("hub4j-test-org"))
-                .findFirst()
-                .get();
+        GHApp app = gitHub.getApp();
+
+        GHAppInstallation appInstallation;
+        if (Set.of(TEST_APP_ID_1, TEST_APP_ID_2, TEST_APP_ID_3).contains(Long.toString(app.getId()))) {
```

Comment:
This switch leads to invalid recordings: I've done it to prevent recording the N installations of my real-world app. However, when re-running the tests without my custom JWK, the test will list of installations, which will match no stub (as we recorded the other branch of the `if`, which was focusing on a dedicated repo.)

This remain useful, however one has to edit the recordings manually (e.g. copy-pasting the installations listing from another test).

@bitwiseman Would you mind granting me access to the test repo/app ? It may be useful on related matters not requiring markplace-listing.